lib/ui/profile/Profile.dart

```diff
@@ -74,7 +74,8 @@ class _Strings {
       'https://sites.google.com/farmsmart.co/farmsmart/home/privacy-policy?authuser=0';
   static final termsOfUseUrl =
       'https://sites.google.com/farmsmart.co/farmsmart/home/privacy-policy?authuser=0';
-  static final shareLink = 'https://play.google.com/store/apps/details?id=co.farmsmart.app';
+  static final shareLink =
+      'https://play.google.com/store/apps/details?id=co.farmsmart.app';
```

Comment:
That has been done in another PR, isn't it?

Reply:
yep It's just format code